config/routes.rb

```diff
@@ -7,4 +7,5 @@
   resources :students
   resources :teachers
   resources :courses 
+  resources :courseloads
```

Comment:
This likely needs to be `:course_loads` with the underscore. When you have a constant with multiple words (e.g. `CourseLoad`) it separates each word by an underscore. You can see this by typing `CourseLoad.to_s.underscore` in the console.